dashboard/app/controllers/home_controller.rb

```diff
@@ -45,13 +45,13 @@ def index
   end
 
   # Signed in: render home page
-  # Signed out: redirect to code.org
+  # Signed out: redirect to sign in
   def home
     if current_user
       init_homepage
       render 'home/index'
     else
-      redirect_to CDO.code_org_url
+      redirect_to '/users/sign_in'
```

Comment:
You can instead call [authenticate_user!](https://github.com/plataformatec/devise#controller-filters-and-helpers) at the top of the method, or in a `before_action` callback. It will ensure you have a logged-in user, and otherwise redirect to sign_in for you. You don't need to do this manually.